crates/bifrost/src/providers/local_loglet/log_state.rs

```diff
@@ -172,19 +212,21 @@ pub fn log_state_partial_merge(
         warn!(key = ?key, "Merge is only supported for log-state");
         return None;
     }
-    let mut merged = LogStateUpdates::with_capacity(operands.len());
+    // assuming one entry per LogStateUpdate variant at most since everything can be merged.
```

Comment:
Maybe add this comment also to the field in `LogStateUpdates` to explain why 3 was chosen.

Reply:
Done